spec/robots/dor_repo/speech_to_text/stt_create_spec.rb

```diff
@@ -20,18 +21,17 @@
   let(:workflow_process) do
     instance_double(Dor::Workflow::Response::Process, lane_id: 'lane1', context: { 'runSpeechToText' => true })
   end
-  let(:job_id) { '1234-5678-0000' }
+  let(:job_id) { "#{bare_druid}-v1" }
 
   before do
     allow(Aws::SQS::Client).to receive(:new).and_return(aws_client)
     allow(Dor::Services::Client).to receive(:object).and_return(dsa_object_client)
     allow(Dor::TextExtraction::SpeechToText).to receive(:new).and_return(stt)
     allow(LyberCore::WorkflowClientFactory).to receive(:build).and_return(workflow_client)
-    allow(SecureRandom).to receive(:uuid).and_return(job_id)
   end
 
   context 'when the message is sent successfully' do
-    let(:message_body) { { id: job_id, druid:, media: ['file1.mov', 'file2.mp3'], options: { model: 'large', max_line_count: 80, beam_size: 10 } }.to_json }
+    let(:message_body) { { id: job_id, druid:, options: { model: 'large', max_line_count: 80, beam_size: 10 } }.to_json }
```

Comment:
and then i think this'll need to get reverted if the job json structure isn't actually changing